src/main/java/com/facebook/openwifirrm/ucentral/WifiScanEntry.java

```diff
@@ -22,6 +26,10 @@ public class WifiScanEntry extends WifiScanEntryResult {
 	 * time reference.
 	 */
 	public long unixTimeMs;
+	public Country country;
+	public QbssLoad qbssLoad;
+	public LocalPowerConstraint localPowerConstraint;
+	public TxPwrInfo txPwrInfo;
```

Comment:
I think I would prefer seeing a separate class/field like `InformationElement ie` to hold all of these as sub-fields, do you agree?

Also, do we actually need equality checks on `WifiScanEntry`? If not then I wouldn't bother putting any of those autogenerated overrides since it's adding a lot of bloat to these files...

Reply:
I don't have much of a preference regarding an `InformationElement` class. My thoughts/questions:
- Note that these classes are already part of one package called `informationelement`
- Would there be any methods in `InformationElement`?
- Probably not a big deal, but the `ht_oper` and `vht_oper` fields in `WifiScanEntryResult` are their separate fields (both in the parsed json and in our data structure), but really, they are also IEs like any other according 802.11. We could not include these two in `InformationElement` since they are in the uCentral defined `WifiscanEntryResult`.

Regarding equality checks, we use them for unit tests. But they may not need to include every field...

Reply:
Also, if we do the new class, maybe it should be `InformationElements` since it is containing multiple IEs, and is not itself an IE?

Reply:
At the very beginning, I also preferred a class like `InformationElements` to include new fields so that it's clear that which ones are ies, but now I think your opinion 3 is strong enough to leave them separate without a umbrella class to hold them.

Reply:
@zhiqiand since you mentioned equality checks, here is a thread that discusses it (and another concern). What are your thoughts?

Reply:
I just want to make sure if the equality checks are used somewhere or I missed it. The equality check I mentioned is the classes under `informationelement` folder. If it is not used then it's better not to add it at this moment.

Reply:
I wanted equality checks for `WifiScanEntry` (because equality checks for that class are used in unit testing). Since these classes are part of `WifiScanEntry` they should also implement equality checks.

However, since these specific classes aren't actually what is tested in the unit tests, I could remove them and it wouldn't cause any problems now (though if we add individual IE testing later, it may cause problems).

Thoughts?

Reply:
Will address in a new PR